src/libs/events.js

```diff
@@ -130,7 +130,8 @@ export const extractWorkspaceDetails = (workspaceObject) => {
   // containing the workspace details.
   const workspaceDetails = 'workspace' in workspaceObject ? workspaceObject.workspace : workspaceObject;
   const { name, namespace, cloudPlatform } = workspaceDetails;
-  const data = { workspaceName: name, workspaceNamespace: namespace };
+  const policies = workspaceObject.policies ?? workspaceDetails.policies;
+  const data = { workspaceName: name, workspaceNamespace: namespace, policies };
```

Comment:
This method is for emitting Mixpanel events. What does it look like in the emitted event if we include this information?

Reply:
(I'm thinking that we might rather emit something simpler like the result of `hasProtectedData`)